yarn.lock

```diff
@@ -2227,23 +2227,6 @@
     ts-dedent "^2.0.0"
     util-deprecate "^1.0.2"
 
-"@storybook/addons@6.5.7":
-  version "6.5.7"
-  resolved "https://registry.yarnpkg.com/@storybook/addons/-/addons-6.5.7.tgz#1c7256f696fb572f7c3c7ccbcb94988372b95cee"
```

Comment:
These changes don't belong into this pull request. Same with the `dump.rdb` file. Please commit only changes related to the issue you are solving and address only one issue per commit.